Week-3/Session-09/JavaScript password strength/script.js

```diff
@@ -0,0 +1,12 @@
+// Your solution goes here
+const isStrongPassword = (password) => {
+	password = password.toString();
+	if (password.length >= 8) {
```

Comment:
create a regex that matches the given condition don't use password.length

Reply:
use this for reference https://www.oreilly.com/library/view/regular-expressions-cookbook/9781449327453/ch04s09.html

Reply:
Fixed